terraform/tools/terragrunt.hcl

```diff
@@ -1,11 +1,13 @@
 terraform {
-  source = "git::https://github.com/BCDevOps/terraform-octk-aws-workload-ecr.git//.?ref=v0.0.3"
```

Comment:
I made the tag v0.0.4 so this can be updated

Reply:
Done